lib/scss_lint/linter/declaration_order.rb

```diff
@@ -18,13 +19,14 @@ def check_order(node)
 
     MESSAGE =
       'Rule sets should be ordered as follows: '\
-      '`@extends`, `@includes` without `@content`, ' \
-      'properties, `@includes` with `@content`, ' \
-      'nested rule sets'.freeze
+      '`composes`, `@extends`, `@includes` without ' \
+      '`@content`,  properties, `@includes` with' \
+      '`@content`, nested rule sets'.freeze
 
     MIXIN_WITH_CONTENT = 'mixin_with_content'.freeze
 
     DECLARATION_ORDER = [
+      Sass::Tree::ComposesNode,
```

Comment:
this is wrong, Sass::Tree does not have a ComposesNode, will need to find out what `composes` is